llvm/test/CodeGen/DirectX/legalize-fneg.ll

```diff
@@ -0,0 +1,23 @@
+; RUN: opt -S -passes='dxil-legalize' -mtriple=dxil-pc-shadermodel6.3-library %s | FileCheck %s
+
+define float @negateF(float %x) {

```

Comment:
were there no tests dependent on  the `DXILPrepare` pass for fneg to fsub? I would expect if their were they would be failing now.

Reply:
I believe no because `check-llvm-codegen-*` targets passed all tests.